src/Compatibility/Core/src/Android/CollectionView/ObservableGroupedSource.cs

```diff
@@ -213,7 +214,7 @@ void UpdateGroupTracking()
 
 			for (int n = 0; n < _groupSource.Count; n++)
 			{
-				var source = ItemsSourceFactory.Create(_groupSource[n] as IEnumerable, this);
+				var source = ItemsSourceFactory.Create(_groupSource[n] as IEnumerable, _groupableItemsView, this);
```

Comment:
Here is an example of apssing in the groupable view itself into the items source collection thing above.